src/components/Sessions/SessionExpired.js

```diff
@@ -0,0 +1,50 @@
+import React, {useContext, useEffect} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Card from 'components/Card';
+import ErrorMessage from 'components/Errors/ErrorMessage';
+import Link from 'components/Link';
+import useSessionTimeout from 'hooks/useSessionTimeout';
+
+import {ConfigContext} from '../../Context';
+import {flagNoActiveSubmission} from '../../data/submissions';
+
+const SessionExpired = () => {
+  const config = useContext(ConfigContext);
+  const [expired] = useSessionTimeout(null, true);
+
+  // codecov:ignore:start
```

Comment:
that didn't seem to work :thinking: 

You can try:

```suggestion
/* istanbul ignore next */
```